packages/shared-models/src/ymodels.ts

```diff
@@ -190,6 +195,24 @@ export class YFile
   public ysource = this.ydoc.getText('source');
 }
 
+/**
+ * These are "templates" that can be used as initial content. All clients will start with the same template which prevents that every joining client will create another "initial" cell.
+ *
+ * Note that initialization must not be done dynamically. Hence you need to create a static update message that doesn't change.
+ *
+ * You may creates templates by creating an empty shared notebook model, insert some content (e.g. an intial code cell), and then encoding the update to base64:
+ *
+ * ```js
+ * template = buffer.toBase64(Y.encodeStateAsUpdateV2(model.ydoc))
+ * ```
+ */
+const yCodeCellTemplate =
+  'AAAGyKjKnwUFAgEEAAcHACcAKAMnPDRjZWxsc3NvdXJjZW1ldGFkYXRhY2VsbF90eXBlaWRleGVjdXRpb25fY291bnRvdXRwdXRzBQYICQIPBwMBAAADAQIAAkECAQcAdgB3BGNvZGV3JGU1MTNlNzk2LTM4NmItNDg3Zi1hNGFjLTQ5MjZkNTcyZGJiNX0AAA==';
```

Comment:
Can we document the decoded version of those base64 templates?

Reply:
The above comment explains how to create these "templates". These are base64-encoded Yjs documents - it's not really possible to decode this in a human-readable comment. However, I extended the above comment and added code that shows how these templates are created.

Reply:
Well, it would still be useful to indicate what is the empty document the user will start with. I guess it is a json payload with a few metadata and a cell list have a single empty cell. Could that json payload be added in the typedoc?

Reply:
Makes sense. Will do.